src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
 
 ### Writing a Test case
+The tag [AllureNUnit] its necessary to generate the JSON files.
+In this exemple, the [Description] tag is set as step test case in the allure-reporter tool.  
+The [AllureTms] tag contains the test case name in Jama.
```

Comment:
I suggest 'The [AllureTms] tag is the test case name defined in Jama

Reply:
changed